build.sbt

```diff
@@ -2,13 +2,13 @@ name := "amqp-client"
 
 organization := "com.kinja"
  
-version := "2.2.2" + (if (RELEASE_BUILD) "" else "-SNAPSHOT")
+version := "2.2.3" + (if (RELEASE_BUILD) "" else "-SNAPSHOT")
```

Comment:
I'm not certain it should be a minor version update — it would mean everything else moves to Akka 2.6, so, shouldn't it be at least the same level of version bump?

Reply:
I agree with @migmit-kinja it's contains breaking change:
- Dropping scala 2.11
- Depending on Akka 2.6 instead of 2.5. (If I'm correct you can't use this with akka 2.5 anymore)

Reply:
Okay, 2.3.0 it is then.